articles/active-directory/develop/scenario-web-api-call-api-app-configuration.md

```diff
@@ -138,6 +138,8 @@ private void AddAccountToCacheFromJwt(IEnumerable<string> scopes, JwtSecurityTok
 }
 ```
 
+You can also see an example of on behalf of flow implementation in [NodeJS and Azure Functions](https://github.com/Azure-Samples/ms-identity-nodejs-webapi-onbehalfof-azurefunctions/blob/master/MiddleTierAPI/MyHttpTrigger/index.js#L61).
```

Comment:
@maliksahil, I would suggest an h2

`## On behalf for Node.JS in azure functions`

Moving forward when we add more langages (Python, Java are coming), we'll use tabbed conceptuals, and we'll include node JS.

If this is just a sentence, there is a riks that we overlook it